### DISCUSSION TITLE
Is there a way to have multi VRF Prefixes?

### DISCUSSION THREAD

**Discussion by andyman212** [netbox-community/netbox#15514] [Q&A]
A lot of the customers I have worked for have prefixes like a /24 and break down into /30s each one with different VRF's, They use VRF for segregation but do not overlap the IP space. So in Netbox currently every time I assign a prefix to a VLAN it moves it out of the tree? Any way to get around this or a planned feature to allow multi-VRF prefixes? Thanks

**Comment by candlerb** [comment]
This has been discussed several times before - see for example #6533, #7608, https://github.com/netbox-community/netbox/discussions/12840#discussioncomment-6124118.

IMO it would be good to separate the concepts of "address namespace" (i.e. domain of uniqueness) from "VRF".
